app/controllers/event_attendances_controller.rb

```diff
@@ -0,0 +1,50 @@
+class EventAttendancesController < ApplicationController
+  layout "site"
+  before_action :authorize_web
+  before_action :set_event_attendance, :only => [:update]
+
+  authorize_resource
+
+  def create
+    attendance = EventAttendance.new(attendance_params)
+    attendance.intention = intention
+    if attendance.save!
```

Comment:
model.save! (with an exclamation mark) raises an error if the save is unsuccessful. So here you can use `attendance.save` (without the exclamation mark) which returns a boolean.

Reply:
Fixed this type of issue in 3 places.